screens/Home.js

```diff
@@ -1,12 +1,36 @@
-import React from "react";
-import { Text, View } from "react-native";
+import React, { useState } from "react";
+import { FlatList, Text, TouchableOpacity, View } from "react-native";
+import Card from "../components/Card";
 import { globalStyles } from "../styles/global";
 
 export default function Home({ navigation }) {
 
+    const [sections, setSections] = useState([
+        { section: 'ALBUMS', key: '1'},
+        { section: 'SONGS', key: '2'},
+        { section: 'ARTISTS', key: '3'}
+    ])
+
     return (
         <View style={globalStyles.container}>
             <Text style={globalStyles.text}>HOME SCREEN</Text>
+            <FlatList
+            data={sections}
+                renderItem={({ item }) => (
+                    <TouchableOpacity onPress={() => {
```

Comment:
There is a new kid in town called [Pressable](https://reactnative.dev/docs/pressable) ;)
```suggestion
                    <Pressable onPress={() => {
```